charts/rancher-istio/0.1.2/charts/certmanager/values.yaml

```diff
@@ -5,20 +5,21 @@
 # restart, DestinationRules can be created using the ACME-signed certificates.
 enabled: false
 replicaCount: 1
-image:
-  repository: rancher/jetstack-cert-manager-controller
-  tag: v0.6.2
+hub: quay.io/jetstack
```

Comment:
Since we need all images to reside in dockerhub, what does changing this to quay.io do? I don't believe we should be updating it.

Reply:
yeah, images need to be updated and that will be under values.yaml in root directory.